samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -298,10 +414,12 @@ int main(void)
 		/* Setup extended advertising data */
 		net_buf_simple_add_le16(&ad_buf, BT_UUID_BROADCAST_AUDIO_VAL);
 		net_buf_simple_add_le24(&ad_buf, broadcast_id);
-		ext_ad.type = BT_DATA_SVC_DATA16;
-		ext_ad.data_len = ad_buf.len;
-		ext_ad.data = ad_buf.data;
-		err = bt_le_ext_adv_set_data(adv, &ext_ad, 1, NULL, 0);
+		ext_ad[0] = (struct bt_data)BT_DATA(BT_DATA_BROADCAST_NAME, CONFIG_BT_DEVICE_NAME,
+					    sizeof(CONFIG_BT_DEVICE_NAME) - 1);
+		ext_ad[1].type = BT_DATA_SVC_DATA16;
+		ext_ad[1].data_len = ad_buf.len;
+		ext_ad[1].data = ad_buf.data;
+		err = bt_le_ext_adv_set_data(adv, ext_ad, 2, NULL, 0);
```

Comment:
Isn't the Broadcast Name only used by the PBP? This sample is a BAP Broadcast Source sample

Reply:
You might be right on this :) - I can remove it if so.

Reply:
I think we should keep this as BAP and as simple as possible. 

I realize that that it won't work the NCS application anymore though, if they expect a PBP Broadcast Name